lib/ice_cube/validations/monthly_by_set_pos.rb

```diff
@@ -56,7 +56,9 @@ def validate(step_time, start_time)
       end
 
       def build_s(builder)
-        builder.piece(:by_set_pos) << by_set_pos
+        # returning empty string for BYSETPOS
+        # as this field is already represented by the BYMONTHDAY
```

Comment:
I guess this is part of the problem with the descriptions. `BYMONTHDAY` is providing the "on the 28th, 29th, and 30th of the month" part, and adding `BYSETPOS` can't (or at least doesn't) override that

Reply:
`BYSETPOS` is not represented easily on the string as it works closely with the other params :-/...